src/component/material-ui/nav-wrapper/index.js

```diff
@@ -40,6 +47,39 @@ class NavWrapper extends Component {
     this.setState({ popoverOpen: !this.state.popoverOpen });
   }
 
+  renderSavedSwitch = (location) => {
+    switch (location) {
+      case '/saved':
+        return (
+          <Link
+            to="/" 
+            href="/"
```

Comment:
Why do we have the `href` here. With `Link` isn't it just supposed to have the `to`?

Reply:
for some reason out eslint required me to put it in, otherwise it would have a red error in our file, which is annoying.
We could change the eslint rule, or just put it in.